internal/safeconfig/safeconfig.go

```diff
@@ -9,8 +9,13 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+type Domain struct {
+	Name string `yaml:"name"`
+	Host string `yaml:"host,omitempty"`
+}
+
 type SafeConfig struct {
-	Domains []string `yaml:"domains"`
+	Domains []Domain `yaml:"domains"`
```

Comment:
this is a breaking change, right?

Reply:
I suppose yes, previously the config file was like:
```
domains:
- google.com
- apple.com
```
after this change it expects something like below:
```
domains:
- name: google.com
- name: apple.com
  host: some.whois.server
- name: github.com
```
so old config wouldn't work with new version :<

Reply:
no biggie, we do a major bump

Reply:
FWIW: made it so it isn't; https://github.com/caarlos0/domain_exporter/commit/ec20d50990ee71c32d6bf241f7a277c4216a15fd